### ISSUE TITLE
Unable to refresh

### ISSUE THREAD

**Issue by filipedelbel** [stansw/vsts-open-in-powerbi#23]
Hello,

I was using the plugin flawlessly until earlier today. No changes were made anywhere but it started showing this error message when I try to refresh data on PowerBI:

"500 - Something went wrong!</div> <div class="details">We've encountered an error and cannot fulfill this request. We will trace the error and get a fix out soon.</div> <div class="reference-data"> <div>5/11/2022 5:43:20 PM (UTC)</div> "

Would this be a known issue? I tried using the plugin in other queries, but I'm having the same response so far.

I appreciate any attention on this, thank you!

**Comment by gabrielviktor** [comment]
Hello,

I'm receiving this error messagens too.

Does anyone know how to proceed?

Thanks

**Comment by vittoneMaxi** [comment]
Hola, me pasa lo mismo. El complemento venia funcionando bien.
![image](https://user-images.githubusercontent.com/105662259/168635484-19526720-e2ee-4c0f-aac6-2bbeaf0118be.png)
![image](https://user-images.githubusercontent.com/105662259/168656421-f1a329a2-749a-4eb6-a54b-1f1fd85a6f2a.png)

**Comment by thcook86** [comment]
My team is getting the same error and cannot refresh our reports. Pasting the full error below. 

Data source error: <pii>Error</pii>: <pii><!DOCTYPE html >
<html>
  <head>
    <title>This operation is not supported for a relative URI.</title>
    <style type="text/css">html {
    height: 100%;
}

body {
    font-family: "-apple-system", BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Helvetica, Ubuntu, Arial, sans-serif, "Apple Color Emoji",
        "Segoe UI Emoji", "Segoe UI Symbol";
    font-size: 14px;
    height: 100%;
    margin: 0;
    background-color: #fff;
}

.container {
    height: 100%;
    display: flex;
    flex-grow: 1;
    flex-direction: column;
}

.header {
    font-size: 14px;
    line-height: 20px;
    padding-right: 40px;
    display: flex;
    box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.08);
}

.header > div,
.header > a {
    margin: 14px 0px 14px 32px;
    text-decoration: none;
    color: rgba(0, 0, 0, 0.55);
}

.header > div:first-child {
    flex-grow: 1;
    margin-left: 14px;
    color: #0078d4;
    display: flex;
}

.header > div:first-child > div {
    margin-left: 14px;
}

.content-container {
    display: flex;
    flex-direction: column;
    justify-content: center;
    margin-top: 126px;
}

.content {
    display: flex;
    justify-content: center;
    align-items: center;
}

.error-info {
    display: flex;
    flex-direction: column;
    width: 500px;
}

.title {
    font-size: 28px;
    font-weight: bold;
    line-height: 40px;
    letter-spacing: -0.04em;
    color: #212121;
    margin-bottom: 16px;
}

.details {
    font-size: 14px;
    line-height: 20px;
    color: #212121;
    margin-bottom: 16px;
}

.details a {
    text-decoration: none;
    color: #0078d4;
}

.reference-data {
    font-size: 12px;
    line-height: 16px;
    color: rgba(0, 0, 0, 0.55);
    margin-bottom: 32px;
}

.logo {
    height: 500px;
    width: 500px;
}

.action-container {
    display: flex;
}

.action {
    background-color: #0078d4;
    color: #ffffff;
    align-items: center;
    border-radius: 2px;
    border: 1px solid transparent;
    cursor: pointer;
    display: inline-flex;
    flex-direction: row;
    flex-shrink: 0;
    font-family: inherit;
    font-size: inherit;
    font-weight: 600;
    justify-content: center;
    outline: none;
    overflow: visible;
    padding: 6px 12px;
    position: relative;
    transition: background 80ms linear;
    text-decoration: none;
}

.action:focus {
    box-shadow: 0 0 0 3px rgba(0, 120, 212, 0.35);
}

.action:hover {
    background-color: rgb(0, 103, 181)
}

.action:active {
    background-color: rgb(0, 91, 161)
}

.secondary-action {
    text-decoration: none;
    color: #0078d4;
    margin-top: 16px;
}

.more-info {
    margin: auto;
    display: flex;
    flex-direction: column;
    padding: 0 40px;
}

.more-info > div:first-child {
    font-size: 15px;
    font-weight: 700;
}
</style>
  </head>
  <body>
    <div class="container">
      <div class="header">
        <div>
          <span>
            <img class="header-logo" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAGBSURBVHgB5VRBUsJAEOxdYuXgQZ4QPSIH8oP4AvUH8gLiQQleOKHiRXyBX+AH5AdgmUq8EX4QT5Iqk3U2BAlViRHhZlelsjXb0+nszgxQhvargWvHxC/BSxmMaeD8EZY7xZXT2F6QKwfpSkOFj2F5z2h7WhGd5UbbngGIU1oZxMhz5SOOn/BQH2RyNAjRWAnevLUQxQZ5NiBQRbGFgPYnyVPhJCxMMFFd5igJyXIG9MVW4lcUJHNGj7DRq83W/oQJLZujoOOaFGilApQsJmDcBotecFufYENIh10sz+WupmNL8B/P60+CO8a/FGRJNwyxI6w6xRxXoapdGgby1n0q5IC64J2qwE8ZPsJ5gIEefOcsCnuUL5gQaFSxygjlPnxqgCBVaBQLSnRo9gkaV5tCtmn02cyfNpYrXZJbdgERy/dZSQOQ4/AEfd3Pv2U1PE9IXLVxf9zER3gIxE2K2TlsG/NQl2ILo0WQ56nsT9E7mq3HxxrYnkEF0iXXQ/Rrl9ntLw/ghshmZ3a8AAAAAElFTkSuQmCC" />
          </span>
          <div>Azure DevOps</div>
        </div>
        <a href="https://go.microsoft.com/fwlink/?LinkID=242573">Service Status</a>
        <a href="https://go.microsoft.com/fwlink/?linkid=2119200&amp;clcid=0x409">Support</a>
        <a href="https://twitter.com/AzureDevOps">@AzureDevOps</a>
      </div>
      <div class="content-container">
        <div class="content">
          <div>
            <img class="logo" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAfQAAAH0CAYAAADL1t+KAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAKChSURBVHgB7f0JtGXJVd8J77gvX85ZmVlZlZk1D6oqVWmiJNS4AUndBoQlJhvaQu4PDwi6V4NtjD8b2gZ5+fNn7MbubtMLhKdeDZKHXr0QuOkGg2g0YAmQBBJSlaZSzaox56ky883v7j5xTgx774hz33s5vnfe/1f18p4hTkScc889/9g7dsRx3EAAAAAA2NCMCAAAAAAbHgg6AAAAMAAg6AAAAMAAgKADAAAAAwCCDgAAAAwACDoAAAAwACDoAAAAwACAoAMAAAADAIIOAAAADAAIOgAAADAAIOgAAADAAICgAwAAAAMAgg4AAAAMAAg6AAAAMAAg6AAAAMAAgKADAAAAAwCCDgAAAAwACDoAAAAwACDoAAAAwACAoAMAAAADAIIOAAAADAAIOgAAADAAIOgAAADAAICgAwAAAAMAgg4AAAAMAAg6AAAAMAAg6AAAAMAAgKADAAAAAwCCDgAAAAwACDoAAAAwACDoAAAAwACAoAMAAAADAIIOAAAADAAIOgAAADAAIOgAAADAAICgAwAAAAMAgg4AAAAMAAg6AAAAMAAg6AAAAMAAgKADAAAAAwCCDgAAAAwACDoAAAAwACDoAAAAwACAoAMAAAADAIIOAAAADAAIOgAAADAAIOgAAADAAICgAwAAAAMAgg4AAAAMAAg6AAAAMAAg6AAAAMAAgKADAAAAAwCCDgAAAAwACDoAAAAwACDoAAAAwACAoAMAAAADAIIOAAAADAAIOgAAADAAIOgAAADAAICgAwAAAAMAgg4AAAAMAAg6AAAAMAAg6AAAAMAAgKADAAAAAwCCDgAAAAwACDoAAAAwACDoAAAAwACAoAMAAAADAIIOAAAADAAIOgAAADAAIOgAAADAAICgAwAAAAMAgg4AAAAMAAg6AAAAMAAg6AAAAMAAgKADAAAAAwCCDgAAAAwACDoAAAAwACDoAAAAwACAoAMAAAADAIIOAAAADAAIOgAAADAAIOgAAADAAICgAwAAAAMAgg4AAAAMAAg6AAAAMAAg6AAAAMAAgKADAAAAAwCCDgAAAAwACDoAAAAwACDoAAAAwACAoAMAAAADAIIOAAAADAAIOgAAADAAIOgAAADAANhCAABwhZlZHNPSmGhxmWl2cbnZ4mh6ytGWkWuWuV3eMT2i6XYdAHAlgKADAC6ZxTHT6ZklOjmz3Ag30ytz3adzXsJdI93cJfQfzTbmeKSjcbPiBX7vdkc3bJ+iG7ZNNcuj5m+KAABrxzHnnxgAAKyEt76fP7tIpxoRPzOz3G7rBJzafxxFq9sLeyPqnD+529x+xgcPx/XwuaU5/MCuKbp1zzTd3Hzu3IqeQQBWAwQdALAi3nXuRfzI+cXGIh+3Au7xH/t3TtH+HVO0uxFe/zAZN1b7QpN+bonp/PyYLi6M27SdaHemupf2VsBbEecs6hSEX4j8LXum6La9W+nu/dMEAOgHgg4A6MUL+VOnFujpU4tNn/i4tbSjNb6z6QN/8OC2RsgdLY879/ty87fUGO1LzWNlKSzPL3eueC/w8XHjJT4+ecbRK8/UuuHjA2nMFKz6TuR9effftLUR9+m28QAA0EDQAQBVnjw5T48dX2jF2Qv4KLjTvaB7cf2GO3e0ZvRSI9itmIflVsgbxW4/OQh8s/1i07c+3+xI7vUg3slKb4W7E3WPFfRotfuy77lxuvnbCmEHQABBB9eNxebWe6V52B/Ygkjn9cTxC0v06JE5OtdY1SNhkfvPUfC1v/WeXW1fdyfeWcCXx3mbXPZ/Pth9pknot6f+dB8cNx5n0absareCPjYW/c5pR68/vJ1edWArAQAg6OA64cX88UYw9k6NWtvszq2IbF4PPH5inr5wdK5d9ravd7G3lnloc3lBP7hrqhXSRSXkwipvPpfTNm+9UxD0rm+9FXSiQqRzv3m3Pbvec397dNXHdH55V2Olf/v9u2n3NljrYHODXwC4jrjWOs9R0eB64cX2Y09faC1zr5btd+Lk9xK/JW6F07vXG8O6DYAbB2vcC/hY9IN3yy7s5yTaEVafei0WrcyNGElvdlyYX6b/8KVX6JGX5wiAzQzGoYPrwnTzxL5jq6OjTb/qvbCsris+Cv1jT1+kmQURvS7c7BpHs4vBAueu33yZo3h3Au+1Vop73Lc8DoPRQ1R7txTHq5Map66EXGzzx/gJabaEW2ahyXNuwXsBxvT5l2fp5fOL9Na7d9Ee3FNgEwKXOwCbmAtezJ+62I4tzyIe3OykhT3u2zrl6OFbtzUK67KLnXO/+XIMkAv7FpuNiyHyPbnOZUCcdKGL/vKxGOLmy9y7bYq2TTsVnBfr5et/ema5HRvvg+a+88E9EHWw6YCgA7BJ8WL+0UbMvYUe+8lbCQwi2QbE+dWRU6Lu0/px535ceOwvT6K+HKx2ERC3GKLgY6Cbp+sLZyXoYxv53vyNmgr52eO2TY2SgMfAvByol5d9PV56ZZHOzY7pux7aQwd2IjYDbB4g6ABsQryYf/jJC61lOzLWro1sT5+uc463FnyzbW8j6j5Azm9dCi74LPCcxqZHkc9D1CpWOlExocxUU87eHaPmU4i4iw0NVmPik7CHup2cWaJnTy/SW+7eSa++eRsBsBlAHzoAm5DPvjgbLHMRaRa6uDn1bEtc2B9C45oPP6zNTxjj52DfsXVEfrzCmPPwtYUwJn1ciHnnWieSfeZiBjnylrmf333UinQcyhZqoZfZ9PNzV7ebd25p54T/1HMzjet9im69AY86MHxgoQOwyXj05Tn6wrE5Nbbc9pOP0jC1/Bn3FVZ7mKu9TTfq9o3NU6Vbjxa4mfo19p2LYLk9jRhPKS+BL5dV33nXPaD7+0di3TcK/EQ2j59YoG9/YBfdtBOiDoYNokYA2ERcmB/To2GceR4CFv5JI8eysLbpXIxCt1Y1CYEO/eBjChPH6AliuvQuC/g4Fx2tdgr5bd/SWfvRI5Dne9dvbxPVFcusPAHbtzh68Oat9HtPXaTz88sEwJCBoAOwifjjF2Y7Xeas4xSmXbXTAeTgtDBSXDjzlIAma1vM9Ob/xmz6xlmNNh/L2WOoC5Tz/eb+XelS6GM9SOZNuUGR6mTThOy3T4/ovgNb6ePPzBAAQwaCDsAmwb8t7YVzC8VkLbI/m0U/ebuNsvVMcnrWwkLmQoC5fec5kZ2LXep4PC5GuPthackaZ2Hpk3AksCmfjJiHxLJB4kXdTxX7iWcvEgBDBYIOwCbhy02/ORez8rnqahJyii53J0Q1WuxZaOXUre3sccxhytawTNpqVsIc3O++z9v3f49DA2FMenY525jo/vIb3KywR2Ldb907TWdml+nFc4sEwBCBoAOwCThyfomONn9dZLj2UzOxUkKOJq603INwyr5xaQ2ndH2Ws7S2o5ueSDUUpp1LddNBdbZcWdfSxa7qGnbEdf8il0ePzBMAQwSCDsAm4KmT812AmZc850o3duVtZ1l8BSlILeyL1jeXlnq3j8XLVWRkOxkXPYchai67zCuNh/iphsJxrgsR1fvbw9+WppA92xx97uVZAmBoQNAB2AQ81/SfszRnPVH55ATuCZc/2Q4xIzHUjKr96t2c7v5YRzE6Xc0ER1qgvbs9NxJIuOi5twzZ2FCeABINlZAfEaUdh3ZvoSdOzNP8kmqqALDhwcBMAK4RS0tLNF5e7j7Hy+17wJeb9eXlpU7EmnW/37Ps9zfLTrihIxzUasv0tNo+vaVbH01NNQI5Cn9T9MrSiHbQfGMBT9Fy04YfByvYjWTwW1BC183k0n5Ei9eP7/a7HXcrLlekO8RlRR13SdrtJMTU5XHmXXn. The exception was raised by the IDbCommand interface. Table: EpicLink.
Cluster URI: WABI-US-NORTH-CENTRAL-H-PRIMARY-redirect.analysis.windows.net
Activity ID: 4edf2b5a-8a67-484f-9cfe-b0f0e8dea77b
Request ID: bcb2530a-d20a-4510-b87e-bfe9fa29877c
Time: 2022-05-17 17:12:55Z

**Comment by jeep** [comment]
We also have this issue.

**Comment by AllenD-MSFT** [comment]
I'm seeing the same problem with PowerBI.com - my PowerBI Desktop .pbix works great in all aspects, including refreshing the data, but trying to refresh data in PowerBI.com has a credential update issue. When I try to update credentials explicitly I'm seeing "This operation is not supported for a relative URI".

But I'm not using this extension. I think the problem is either with PowerBI.com or Azure DevOps itself.

**Comment by gabrielviktor** [comment]
Hey guys, for us is working now. For you too?

Thanks

**Comment by filipedelbel** [comment]
Yes! Just refreshed, thanks for sharing!

**Comment by thcook86** [comment]
Yes, our dashboards could refresh as well. Ours didnt work an hour ago but the refreshes did go through just now. Does anyone have any idea what the fix was?

**Comment by AllenD-MSFT** [comment]
Yep, working for me too. I believe the problem was with a recent change to AzDO.
